CMakeLists.txt

```diff
@@ -353,6 +364,7 @@ get_property(TOPT GLOBAL PROPERTY TOPT)
 set_ifndef(  TOPT -Wl,-T) # clang doesn't pick -T for some reason and complains,
                           # while -Wl,-T works for both, gcc and clang
 
+# mwdt should have custom linker script, or do some speical handling in linker.ld
```

Comment:
mwdt? Is this a TODO?

Reply:
it's a hint for me, alike TODO,  i will clean them,